tensorflow/core/kernels/map_kernels.h

```diff
@@ -106,37 +104,38 @@ class TensorMapSize : public OpKernel {
   }
 };
 
-class TensorMapInsert : public OpKernel {
+class TensorMapLookup : public OpKernel {
  public:
-  explicit TensorMapInsert(OpKernelConstruction* c) : OpKernel(c) {}
-  ~TensorMapInsert() override {}
+  explicit TensorMapLookup(OpKernelConstruction* c) : OpKernel(c) {}
+  ~TensorMapLookup() override {}
 
   void Compute(OpKernelContext* c) override {
     const TensorKey& key = c->input(1);
-    const Tensor& value = c->input(2);
     const TensorMap* m = nullptr;
```

Comment:
Please eschew single letter variable names and abbreviations.  Would you please do

s/m/map/
s/c/context/

everywhere?